internal/pkg/cli/interfaces.go

```diff
@@ -6,6 +6,7 @@ package cli
 import (
 	"context"
 	"encoding"
+	sdkcloudformation "github.com/aws/aws-sdk-go/service/cloudformation"
```

Comment:
We will be able to remove this dependency from `cli` package after we have `cli/deploy/pipeline.go`, is that right?

Reply:
Yeah, the dependency will be moved down into the `clideploy` package and disappear from `cli`.